src/com/untamedears/PrisonPearl/CombatTagListener.java

```diff
@@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
         if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
             return;
         }
-        UUID plruuid = event.getPlayerUUID();
+        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
```

Comment:
good idea,

want me to amend the commit?

On Fri, Feb 27, 2015 at 8:46 AM, rourke750 <notifications@github.com> wrote:

> In src/com/untamedears/PrisonPearl/CombatTagListener.java
> <https://github.com/Civcraft/PrisonPearl/pull/34#discussion_r25510466>:
>
> > @@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
> >          if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
> >              return;
> >          }
> > -        UUID plruuid = event.getPlayerUUID();
> > +        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
>
> I should take down the combat tag off our jenkins page as we no longer
> depend on our custom implementation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Civcraft/PrisonPearl/pull/34/files#r25510466>.
>